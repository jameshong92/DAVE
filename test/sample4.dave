// test data conversion
int main() {
    int a = 83;
    double b = int(a);
    print(b);
    b = 63.23;
    a = int(b+1.5);
    print (a);
    string c = "new";
    bool e = bool(c);
    print(c);
    print(bool(""));
}
