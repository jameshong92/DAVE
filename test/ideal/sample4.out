83.0
64
1
0
